ansible/roles/operator-pipeline/templates/openshift/tasks/verify-ci-results.yml

```
@@ -16,12 +17,21 @@ spec:
     - name: verify-ci-results
       image: registry.access.redhat.com/ubi8-minimal
       workingDir: $(workspaces.results.path)
+      env:
+        - name: PREFLIGHT_MIN_VERSION
+        - value: $(params.preflight_min_version)
       script: |
         #! /usr/bin/env bash
-        # Did the tests pass?
+
         # did preflight/ CI have the latest version?
-        # If results exists, store them in the workspace
-        # In case of failure- comment on the PR with tailored support link
+        PREFLIGHT_VERSION=$(cat PREFLIGHT_RESULTS | jq '.test_library.version' | grep .)
```

```suggestion
        PREFLIGHT_VERSION=$(cat $PREFLIGHT_RESULTS | jq '.test_library.version' | grep .)
```
Missing the `$` meant that you were running cat on a file literally called PREFLIGHT_RESULTS. But also, that var doesn't seem to be defined anywhere?

---

Updated. I forgot to add the variable assignment here.